src/Acuminator/Acuminator.Vsix/AcuminatorVSPackage.cs

```diff
@@ -95,6 +99,12 @@ public GeneralOptionsPage GeneralOptionsPage
             }
         }
 
+		internal AcuminatorLogger AcuminatorLogger
```

Comment:
I think we should consider a possibility of using MEF and Services to register & obtain a logger instance ([one](https://docs.microsoft.com/en-us/visualstudio/extensibility/how-to-get-a-service), [two](https://docs.microsoft.com/en-us/visualstudio/extensibility/how-to-provide-a-service), [three](https://www.c-sharpcorner.com/UploadFile/20c06b/using-mef-to-setup-dependency-injection/))